### DISCUSSION TITLE
measurement_display

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#69] [Announcements]
# measurement_display

本书在 图形化显示数据 一节，详细介绍了 LabVIEW 显示波形数据的方法。这一节，主要介绍在显示测设测量数据中的一些特殊方法。

https://lv.qizhen.xyz/measurement_display

**Comment by JoweZz** [comment]
我在生产者循环通过网口发送数据的速率大概1200w字节一秒，消费者循环中的数据处理和显示完全跟不上这个速度怎么办？

**Reply by ruanqizhen** [reply]
可能需要重新设计一下消费者循环的功能。对于显示来说，人眼的反应速度非常慢，没必要显示所有收到的数据。应该减少显示的数据量以及刷新频率。至于数据处理，如果算力不足以支持实时处理输入的数据量，可以先把数据都保存下来，然后线下处理。
